methods.tex

```diff
@@ -147,17 +143,15 @@ \subsection{Nuclear Deployment in the \gls{EU}}
   G = \text{Growth Trajectory  } [-] 
  \]
  \[
- C_{i} = \text{Nuclear Capacity in Year i  } [\gls{MWe}]
+ C_{i} = \text{Nuclear Capacity in Year i  } [\gls{MWe}].
  \]
 
 The growth trajectory and specific plan of each nation in the \gls{EU} 
-is listed in Table \ref{tab:eu_growth}. Meanwhile, \cref{fig:eu_pow} displays the
-timeseries of installed capacity in \gls{EU} nations.
-
+is listed in Table \ref{tab:eu_growth}.  
 
 \begin{table}[h]
     \centering
-    \caption {Future Nuclear Programs of \gls{EU} Nations \cite{world_nuclear_association_nuclear_2017}}
+    \caption{Projected nuclear deployment strategies of \gls{EU} nations \cite{world_nuclear_association_nuclear_2017}}
```

Comment:
I think `deployment` should be replaced with something that also includes the connotation of `decommissioning`. Maybe just `Projected nuclear trajectories of EU nations`?

Reply:
How about "Projected nuclear power strategies" ?

Reply:
Sounds good! thanks